server/utils/index.js

```diff
@@ -1,50 +1,26 @@
-const { path, pathOr, isNil, isEmpty } = require('ramda');
-const defaultConfig = require('../config');
+const { isNil, isEmpty } = require('ramda');
+const defaultEstablishmentData = require('../content/establishmentData');
 
 const isEmptyResponse = val => isEmpty(val) || isNil(val);
 
-function getEstablishmentId(name, config = defaultConfig) {
-  return Object.keys(config.establishments).reduce(
-    (matchingEstablishmentId, establishmentId) =>
-      config.establishments[establishmentId].name === name
-        ? parseInt(establishmentId, 10)
-        : matchingEstablishmentId,
-    0,
-  );
-}
+const getEstablishmentId = (establishmentName, establishmentData = defaultEstablishmentData) => 
+parseInt(Object.keys(establishmentData).find( id => establishmentData?.[id]?.name === establishmentName), 10)
 
-function getEstablishmentName(id, config = defaultConfig) {
-  return path(['establishments', id, 'name'], config);
-}
+const getEstablishmentName = (id, establishmentData = defaultEstablishmentData) => establishmentData?.[id]?.name
```

Comment:
nice! this was on that doc but we didn't cover it - remove use of ramda where js has overtaken it